tests/testthat/sql/arrange-single.txt

```diff
@@ -35,41 +34,51 @@ ORDER BY `a`
 > # complex mutate
 > lf %>% arrange(a) %>% mutate(a = b) %>% arrange(a)
```

Comment:
Why do we have so many cases here? They all seem rather similar.

Reply:
Reduced.